### ISSUE TITLE
Paratest doesn't recognise class defined in test directory

### ISSUE THREAD

**Issue by mihhac** [brianium/paratest#236]
HI there, thanks for this great tool!

I stuck with the problem. I have base class defined in test directory

// tests/Foo.php
class Foo extends PHPUnit_Framework_TestCase
{
}

// tests/Bar.php
class Bar extends Foo
{
}

// composer.json
"autoload-dev": {
        "psr-4": {
                  "Test\\": "tests"
         }
}

// phpunit.xml is configured to use /vendor/autoload.php

If I run phpunit only, than it will work. But if I try to execute ./vendor/bin/paratest it will report Fatal exception "Class Foo not found".

I've tried with ./vendor/bin/paratest --bootstrap=./vendor/autoload.php I got the same response.

Do you have solution for this problem?

Best!

**Comment by julianseeger** [comment]
Hm... Are both classes in the namespace `Test` ?

**Comment by mihhac** [comment]
HI @julianseeger, thanks for quick reply.

Yes they are in the same namespace (Test). As I said this will work with phpunit, so there is no problem with namespaces, autoloader or anything similar.

Best!

**Comment by julianseeger** [comment]
I've setup a working example with your code: https://github.com/julianseeger/bp-236
Travis-Build working: https://travis-ci.org/julianseeger/bp-236

The only chances I made are adding a test to Bar and rename it to BarTest (for the test audodetection). It also works with "Bar" only but then you have to explicitly specify the file as a test.
Feel free to fork and modify the example to make it more like your project until it fails. My guess would be some namespace issues or some in- or excludes in the phpunit configuration that might be ignored by paratest ... but we have too few information to know. So I suggest that you create a working (in this case failing) example out of that repo

**Comment by mihhac** [comment]
Hi @julianseeger,

Thanks a lot for your support! I figured out what cause the issue, and it is due to my project.  Missing classes were in directory Something/Foo, but on top of the class it was written Something/Bar. Therefore paratest could find parent class, because namespace was messed up. 

Somehow phpunit could resolve it, but thanks to parates I could find this bug.

I tried reconstruct this situation on your playground repository, but without success.

Best!
